examples/flax/speech-recognition/run_flax_speech_recognition_seq2seq.py

```diff
@@ -439,6 +439,7 @@ def main():
             data_args.dataset_config_name,
             split=data_args.train_split_name,
             cache_dir=data_args.dataset_cache_dir,
+            num_proc=data_args.preprocessing_num_workers,
```

Comment:
@amyeroberts, would this be a welcome change to the example training script, or is this something users should adapt on their own?

Reply:
I think this is a good addition - it doesn't complicate the script and the argument `data_args.preprocessing_num_works` already exists and is used in the file. In fact - as I a user I would have expected this to be used here!